tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/TsFileMetadata.java

```diff
@@ -66,18 +64,15 @@
   public static TsFileMetadata deserializeFrom(ByteBuffer buffer) {
     TsFileMetadata fileMetaData = new TsFileMetadata();
 
-    // deviceMetadataIndex
+    // metadataIndex
     int deviceNum = ReadWriteIOUtils.readInt(buffer);
-    Map<String, Pair<Long, Integer>> deviceMetaDataMap = new HashMap<>();
+    List<MetadataIndexNode> deviceMetaDataList = new ArrayList<>();
```

Comment:
Should we rename `deviceMetaDataList` to `metadataIndexList`?

Reply:
Thanks, I have checked all the `deviceMetaDataList` names  and renamed them.